calculator/parser.py

```diff
@@ -0,0 +1,132 @@
+BINARY_OPERATORS = ["+", "-", "*", "/"]
+
+
+def binary_operator(num_1, num_2, op):
+    """
+    Evaluates and acts on different binary operators
+
+    Parameters
+    ----------
+    num_1 : integer
+        The first number to be operated on
+    
+    num_2 : integer
+        The second number to be operated on
+    
+    op : char
+        The operator that is going to be applied to the numbers
+
+    Returns
+    -------
+    integer : The final answer after after the nums are operated on
+    """
+
+    if op == "+":
+        return num_1 + num_2
+
+    elif op == "-":
+        return num_2 - num_1
+
+    elif op == "*":
+        return num_1 * num_2
+
+    elif op == "/":
+        return num_2 / num_1
+
+
+def push(e, stack):
+    """
+    Pushes the element (e) to the given stack
```

Comment:
You should indicate in the docstring that this method mutates the original list so callers won't be surprised.